README.md

```diff
@@ -1,35 +1,61 @@
-
-### Prometheus
-[Prometheus](https://prometheus.io/) is a very popular time-series, metrics database that is used extensively both inside of as well as outside of Kubernetes deployments.
-
-### Grafana
-[Grafana](https://grafana.com/) is the de facto standard for dashboards.
-
-## Repairs
-[Reaper](http://cassandra-reaper.io/) is used to schedule and manage repairs in Cassandra.
-
-## Backup & Restore
-
-
-# Implementation
-K8ssandra is essentially an aggregation of several components that  together comprise the stack described above.
-
-* [cass-operator](https://github.com/datastax/cass-operator)
-* [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator)
-* [grafana-operator](https://github.com/integr8ly/grafana-operator)
-* [reaper-operator](https://github.com/thelastpickle/reaper-operator)
-* [helm](https://helm.sh)
\ No newline at end of file
+[K8ssandra](https://k8ssandra.io/) is a simple to manage, production ready,
+distribution of [Apache Cassandra](https://cassandra.apache.org/) and
+[Stargate](https://stargate.io/). It is built on a foundation of rock solid
+open-source projects covering both the transactional and operational aspects of
+Cassandra deployments. This project is distributed as a collection of
+[Helm](https://helm.sh/) charts. Feel free to fork the repo and contribute. If
+you're looking to install K8ssandra head over to the [Getting
+Started](https://k8ssandra.io/docs/getting-started/) guide.
+
+## Components
+K8ssandra is composed of a number of sub-charts each representing a component in
+the K8ssandra stack. The default installation is focused on developer
+deployments with all of the features enabled and configured for running with a
+minimal number of resources. Many of these components may be deployed
+independently in a centralized fashion. Below is a list of the components in the
+K8ssandra stack with links to the appropriate projects.
+
+### Apache Cassandra
+K8ssandra packages and deploys [Apache Cassandra](https://cassandra.apache.org/)
+via the [cass-operator](https://github.com/datastax/cass-operator) project. Each
+Cassandra container has the [Management API for Apache Cassandra
+(MAAC)](https://github.com/datastax/management-api-for-apache-cassandra) and
+[Metrics Collector for Apache
+Cassandra(MCAC)](https://github.com/datastax/metric-collector-for-apache-cassandra)
+pre-installed and configured to come up automatically.
+
+### Stargate
+[Stargate](https://stargate.io/) provides a collection of horizontally scalable
+API endpoints for interacting with Cassandra databases. Developers may leverage
+REST and GraphQL alongside the traditional CQL interfaces. With Stargate
+operations teams gain the ability to independently scale coordination (Stargate)
+and data (Cassandra) layers. In some use-cases this has resulted a lower TCO and
```

Comment:
Missing the `in` from `result a lower TCO`

Reply:
Actually @bradfordcp the few things I've noticed are trivial typos... let me just push a commit to the branch to fix them so you don't have to dig back into it just for this stuff.